gptresearcher/actions/web_scrape.py

```diff
@@ -123,14 +129,14 @@ def scrape_text_with_selenium(url: str) -> tuple[WebDriver, str]:
         "firefox": FirefoxOptions,
     }
 
-    options = options_available[CFG.selenium_web_browser]()
```

Comment:
@gregdrizz not sure this is who I would go. It's weird passing an object to a util function. The util functions should be redundant of objects such as GPTResearcher. A better approach is to call this functions with the direct params instead of passing an object. So for example the scrap function would get params such as "user_agent", "web_browser", etc. You can also consider writing a higher order method within gpt_researcher that passes the right params and calls this functions. Your call.